### ISSUE TITLE
Show & Tell: ChartScanAI - Using YOLOv8 for financial chart pattern detection

### ISSUE THREAD

**Issue by xiaohoumin3-alt** [ultralytics/ultralytics#24880]
Hi Ultralytics team,

I'm using YOLOv8 for an interesting use case: detecting Buy/Sell patterns on financial candlestick charts.

The system:
1. Generates candlestick charts from MT5 data
2. Runs YOLOv8 to detect trading signals
3. Aggregates across 5 timeframes (M5-M4)
4. Auto-posts results to Telegram

GitHub: https://github.com/xiaoumin3-alt/ChartScanAI

This could be useful for traders who want AI-powered chart analysis. The model was fine-tuned on XAUUSD patterns.

Happy to contribute this as a use case example!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @xiaohoumin3-alt, thank you for sharing this interesting YOLO use case with Ultralytics 🚀📈! Financial chart pattern detection and multi-timeframe signal aggregation sound like a great **Show & Tell** application. This is an automated response to help get things started, and an Ultralytics engineer will also assist soon.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python) and [CLI](https://docs.ultralytics.com/usage/cli) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example) to help us debug it.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips).

For a Show & Tell like this, it would be helpful if you could share a bit more context such as:
-Which Ultralytics model/version you used
-How the dataset was labeled for Buy/Sell patterns
-Example inference outputs or visual results
-Any training metrics, validation results, or deployment notes
-How others in the community might reproduce or learn from the workflow

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

-**Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
-**Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial)
-**Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial)
-**Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes) and [Tasks](https://docs.ultralytics.com/tasks) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by glenn-jocher** [comment]
Thanks for sharing this use case.

**Comment by glenn-jocher** [comment]
Closing this acknowledged [show-and-tell submission](https://github.com/ultralytics/ultralytics/issues/24880#issuecomment-4762717895). The issue shares an external use case and does not identify a reproducible Ultralytics defect or a scoped change to this repository. The project link and discussion remain available here; this disposition does not validate the application’s trading performance.
